org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/org.OpenT2T.Sample.SuperPopular.Thermostat-2.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+    <description>OpenT2T SuperPopular Thermostat interface</description>
+
+    <property name="PowerStatus" type="s" access="readwrite">
```

Comment:
DeviceAccessor currently supports both property accessors and actual JavaScript properties. Note that in JavaScript you can define code for property getters/setters, that are still invoked using property syntax, using [Object.defineProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty). And TypeScript has a nicer syntax for it, more like C# properties. Supporting that style of properties will make it easier for translator developers to use simple properties for things like device attributes and options.